examples/cd_hit_dup.json

```diff
@@ -0,0 +1,22 @@
+{
+  "output_dir_s3": "s3://idseq-samples-development/markazhang/samples_17958/cd_hit_dup",
```

Comment:
These example dags are nice for iterating but I don't think you should check them in. (If anything, write a unit test?)
Also, you should censor the S3 path or at least the id in there...

Reply:
Okay I'll remove the examples. I looked briefly into the unit tests and they rely on `examples/paired_dag.json` which also uses an S3 path to store the test files. I tried to run the unit tests and they all failed...we should discuss how high-priority reviving them is.